tripal_chado/includes/TripalFields/data__protein_sequence/data__protein_sequence.inc

```diff
@@ -93,7 +93,7 @@ class data__protein_sequence extends ChadoField {
       WHERE
         FR.object_id = :feature_id and
         CVT.name = 'polypeptide' and
-        RCVT.name = 'derives_from'
+        (RCVT.name = 'derives_from' or RCVT.name = 'part_of')
```

Comment:
`IN('derives_from', 'part_of')` is actually faster IF the the column is indexed (see: https://stackoverflow.com/questions/3074713/in-vs-or-in-the-sql-where-clause).   I don't know off the top of my head if it is or not.

(I also think it looks cleaner this way but i wouldnt be annoying about that :P)

Reply:
It is: https://chado.readthedocs.io/en/rtd/_static/schemaspy_integration/tables/cvterm.html#Indexes

I therefore recommend using an IN clause instead.

Reply:
```suggestion
RCVT.name  IN ('derives_from', 'part_of')
```